src/workerd/api/url-standard.h

```diff
@@ -287,14 +287,7 @@ class URL: public jsg::Object {
   jsg::UsvString getHash();
   void setHash(jsg::UsvString value);
 
-  inline jsg::Ref<URLSearchParams> getSearchParams(v8::Isolate* isolate) {
-    KJ_IF_MAYBE(searchParams, maybeSearchParams) {
-      return searchParams->addRef();
-    }
-    auto searchParams = jsg::alloc<URLSearchParams>(inner.query, *this);
-    maybeSearchParams = searchParams.addRef();
-    return kj::mv(searchParams);
-  }
+  jsg::Ref<URLSearchParams> getSearchParams();
```

Comment:
This one was just a drive by while I was in here. There's no reason for this one method to be inlined.